walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +274,186 @@ public void notEnoughRoom() {
 
```

Comment:
Very detailed and comprehensive test.

Can you try creating a setUp() method where you do a lot of the work that's common to all your tests?  For example, where you're adding to the Event collection?